snippets/python/Diamond_Square.md

```diff
@@ -0,0 +1,49 @@
+# Diamond Square
+
+## Description
+A Program to print the Diamond Sqaure Pattern, number of rows is given as input.
+
+## Code
+```python
+# The Diamond Pattern
+n=int(input("Enter number of Rows: "))
+
+# The Diamond Pattern
+n=int(input("Enter number of Rows: "))
+
+# The Top Pyramid
+for i in range(n):
+    space="  "*(n-i-1)
+    star=(2*i)*'* '+'*'
+    print(space+star)
+    
+# The bottom Pyramid
+for i in range(n):
+    space = ("    " if i!=0 else "  ")+"  "*(i-1)
```

Comment:
The code looks a bit messy. You can reformat the code with some formatter like black, etc. That will make the code more readable!

Reply:
I was unaware of code formatter , thanks for the suggestion.